src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/Runtime/General/Helpers.cs

```diff
@@ -113,7 +113,7 @@ public static string EscapeTypeNameIdentifier(this string identifier)
             // We're avoiding calling into MemoryExtensions here as it has paths that lead to reflection,
```

Comment:
This comment says that this code is intentionally avoiding `MemoryExtensions` (ie `AsSpan`). If this change passes CI, it is no longer applicable and it should be deleted.